contracts/RSKOwner.sol

```diff
@@ -47,7 +47,7 @@ contract RSKOwner is ERC721, Ownable {
     }
 
     function ownerOf(uint256 tokenId) public view returns (address) {
-        require(expirationTime[tokenId] > now, "Owner query for expired name");
+        require(!_exists(tokenId) || expirationTime[tokenId] > now, "Owner query for expired name");
```

Comment:
This is already checked on `super.ownerOf`

Reply:
I think we should show show one exception when the token is expired and another one when the token doesn't exist. With this addition, in case the token doesn't exist, it will pass this require but will fail in the `super.ownerOf` with the proper exception